Kernel/Arch/i386/CPU.h

```diff
@@ -1087,7 +1088,7 @@ struct TrapFrame {
     TrapFrame& operator=(TrapFrame&&) = delete;
 };
 
-#define TRAP_FRAME_SIZE (2 * 4)
+#define TRAP_FRAME_SIZE (3 * 4)
```

Comment:
Any reason this can't be `sizeof(TrapFrame)`

Reply:
Oh nvm, I see how it's used in the inline assembly :)

Reply:
Isn't there a way to provide an integer value through `"i"` or something like that? So there may be a better way ;-)